lib/sqlalchemy/orm/session.py

```diff
@@ -1633,7 +1633,8 @@ def __init__(
             .. tip:: When using SQLite, the SQLite driver included through
                Python 3.11 does not handle SAVEPOINTs correctly in all cases
                without workarounds. See the section
```

Comment:
**mike bayer** (zzzeek) wrote:

"sections"

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4907